docs/src/modules/components/MarkdownElement.js

```diff
@@ -7,17 +7,6 @@ import { withStyles } from '@material-ui/core/styles';
 import textToHash from 'docs/src/modules/utils/textToHash';
 import prism from 'docs/src/modules/components/prism';
 
-// Monkey patch to preserve non-breaking spaces
```

Comment:
They no longer replace non-breaking spaces (https://github.com/markedjs/marked/blob/4af69d33434f47b85630ea435a0ddeccea244838/lib/marked.js#L553) but they also does not replace the newline symbol `\u2424` with newline escape sequences. It's unclear if this affects us.

Reply:
No clue, the original motivation was #10252. Probably worth trying +1.